Zero-Gravity-Store-React/src/Components/Navigation/Navigation.js

```diff
@@ -1,11 +1,12 @@
-import "./Navigation.css"
+import "./Navigation.css";
 import logo from "../../Asserts/Images/hero-logo.png";
-import {Link} from "react-router-dom";
+import { Link } from "react-router-dom";
 import { useState, useEffect } from "react";
-import {  useFliters } from "../../Contexts/Filter-context";
-export  function Navigation() {
-
-  const [search,setSearch]= useState("");
+import { useFliters } from "../../Contexts/Filter-context";
+import { useAuth } from "../../Contexts/Auth-context";
```

Comment:
Just a suggestion, react packages and tools should be imported first, then CSS files and components from other folders.